compiler/src/iree/compiler/Codegen/Common/GPU/test/BUILD.bazel

```
@@ -34,6 +34,7 @@ iree_lit_test_suite(
             "gpu_promote_matmul_operands.mlir",
             "gpu_reorder_workgroups_static.mlir",
             "gpu_reorder_workgroups.mlir",
+            "gpu_reuse_shared_memroy_allocs.mlir",
```

Please fix the filename typo before landing
```suggestion
            "gpu_reuse_shared_memory_allocs.mlir",
```